ydb/tests/tools/kqprun/start_connector.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -eux
+
+function cleanup {
+    rm ./connector_config.yaml
+}
+trap cleanup EXIT
+
+if [ $# -le 0 ]; then
+    echo "Please provide connector port (value of Generic::Connector::Endpoint::Port in config)"
+    exit -1
+fi
+
+if [ $# -gt 2 ]; then
+    echo "Too many arguments"
+    exit -1
+fi
+
+if netstat -tuln | grep ":$1"; then
+    echo "Can not start connector on port $1, port already in use"
+    exit -1
+fi
+
+SCRIPT_DIR=$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)
+
+cp ${2:-"$SCRIPT_DIR/configuration/connector_config.yaml"} ./connector_config.yaml
+sed -i "s/\${TARGET_PORT}/$1/g" ./connector_config.yaml
```

Comment:
... or drop `cp` line above and
`sed "s/.../" "${2:-...}" > ./connector_config.yaml`
in single step